src/calculateRentalCost.js

```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let result = 40 * days;
```

Comment:
Consider using a constant to define the daily rental cost instead of using a literal value '40'.